src/consts/config.ts

```diff
@@ -17,7 +15,3 @@ export const config: Config = Object.freeze({
   apiUrl: 'https://explorer4.hasura.app/v1/graphql',
   explorerApiKeys,
 });
-
-// Based on https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/mainnet3/agent.ts
-// Based on https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/testnet4/agent.ts
-export const unscrapedEvmChains = [CoreChain.proteustestnet, CoreChain.sei, CoreChain.viction];
```

Comment:
there might still be an entry in the DB for unscraped chains, will that have a negative impact?

Reply:
Ah maybe. I checked and proteustestnet is not in the domains table but sei and viction are. Why don't we scrape those? The impact would be that they should up in the chain list but show no messages when you filter to them. I can restore this list but it would be great if we can kill it eventually

Reply:
for sei, concerns about [pointer contracts](https://www.docs.sei.io/dev-interoperability/pointer-contracts) and how the scraper might not gracefully exit if it hits an issue there

for viction, their RPC non-compliance breaks scraping

will see about removing from the DB and once that's good we can come back and kill this